packages/medusa/src/api-v2/admin/price-lists/query-config.ts

```diff
@@ -0,0 +1,63 @@
+import { prefixArrayItems } from "@medusajs/utils"
+import { defaultAdminProductsVariantFields } from "../products/query-config"
+
+export const defaultAdminPriceListFields = [
+  "id",
+  "created_at",
+  "deleted_at",
+  "description",
+  "ends_at",
+  "id",
+  "title",
+  "starts_at",
+  "status",
+  "type",
+  "updated_at",
+  "price_list_rules.price_list_rule_values.value",
+  "price_list_rules.rule_type.rule_attribute",
+  "price_set_money_amounts.money_amount.id",
+  "price_set_money_amounts.money_amount.currency_code",
+  "price_set_money_amounts.money_amount.amount",
+  "price_set_money_amounts.money_amount.min_quantity",
+  "price_set_money_amounts.money_amount.max_quantity",
+  "price_set_money_amounts.money_amount.created_at",
+  "price_set_money_amounts.money_amount.deleted_at",
+  "price_set_money_amounts.money_amount.updated_at",
+  "price_set_money_amounts.price_rules.value",
+  "price_set_money_amounts.price_rules.rule_type.rule_attribute",
+  ...prefixArrayItems(
+    defaultAdminProductsVariantFields,
+    "price_set_money_amounts.price_set.variant_link.variant."
+  ),
```

Comment:
thought: Should we add a `fieldAlias` to the link definition, so we don't have to query this `variant_link` directly?

Something like:
```
{
      serviceName: Modules.PRICING,
      relationship: {
        serviceName: LINKS.ProductVariantPriceSet,
        primaryKey: "price_set_id",
        foreignKey: "id",
        alias: "variant_link",
      },
      fieldAlias: {
        variant: "variant_link.variant"
      },
    },
```

This would reduce the field to `price_set_money_amounts.price_set.variant`

Reply:
Also, maybe we should review the UI to see if the variants on money amount are still required. It seems more appropriate to do a separate query in case we need them.

Reply:
We added it to make it compatible with the existing API response, back when enabling modules was still backwards compatible. 

We can definitely remove most of it and just keep the ID and query the variants API on the FE.